src/components/icons.js

```diff
@@ -33,8 +33,8 @@ import homeOutline from '@iconify-icons/mdi/home-outline'
 import informationIcon from '@iconify-icons/mdi/information-outline'
 import launch from '@iconify-icons/mdi/launch'
```

Comment:
I am obsessive about putting stuff like this in alphabetical order, but stopped myself elsewhere from polluting this code review with this. I might do a code review for files in this feature which is _just_ alphabetization to spare you as a code reviewer from having to see it in feature PRs.